classwork/classwork4.1/PizzaBuilder.cpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+
+#include "Ingredient.cpp"
+
+// Abstract PizzaBuilder class to construct pizzas
+class PizzaBuilder {
```

Comment:
builder class should have a method buildPizza or something like that to get constructed object. The main idea here is that you move part of the functions related to creation of objects from its API to builder which allows you to construct different object while Pizza class will be clean from that extra routine.
PizzaBuilder should allow to setup different components to construct an object and then you can request that object. After that you can start construction of a new Pizza using the same builder